plugins/woocommerce/includes/emails/class-wc-email-customer-pos-completed-order.php

```diff
@@ -205,6 +207,7 @@ private function add_pos_customizations() {
 		private function remove_pos_customizations() {
 			// Remove actions and filters after generating content to avoid affecting other emails.
 			remove_action( 'woocommerce_order_item_meta_start', array( $this, 'add_unit_price' ), 10 );
+			remove_filter( 'woocommerce_get_order_item_totals', array( $this, 'order_item_totals' ), 10 );
```

Comment:
It might be nice to add a test around this to make sure that the filter is in fact removed and extra info is not bleeding into other order item totals outside these emails.

Reply:
Great suggestion, I added a test case for the POS and non-POS email content in https://github.com/woocommerce/woocommerce/pull/57734/commits/2659f9effe1d6cad93cec9d712e7b4238d5e7380. Please feel free to suggest other ways to unit test this.